infra/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+    container_name: cometbft-node0
+    image: "cometbft/cometbft:v0.37.x"
+    ports:
+      - "26656-26657:26656-26657"
+    environment:
+      - ID=0
+      - LOG=${LOG:-cometbft-node0.log}
+      - CMT_PEX=true
+      - CMT_PROXY_APP=tcp://fendermint-node0:26658
+      - CMT_PERSISTENT_PEERS="${CMT_PERSISTENT_PEERS}"
+    volumes:
+      - ${HOME}/.ipc-testnet/node0/cometbft:/cometbft
+    healthcheck:
+      test: curl --fail http://localhost:26657 || exit 1
+      interval: 8s
+      timeout: 10s
+      retries: 20
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.2
+
+  cometbft-node1:
+    container_name: cometbft-node1
+    image: "cometbft/cometbft:v0.37.x"
+    ports:
+      - "26659-26660:26656-26657"
+    environment:
+      - ID=1
+      - LOG=${LOG:-cometbft-node1.log}
+      - CMT_PEX=true
+      - CMT_PERSISTENT_PEERS="${CMT_PERSISTENT_PEERS}"
+      - CMT_PROXY_APP=tcp://fendermint-node1:26658
+    volumes:
+      - ${HOME}/.ipc-testnet/node1/cometbft:/cometbft
+    healthcheck:
+      test: curl --fail http://localhost:26657 || exit 1
+      interval: 8s
+      timeout: 10s
+      retries: 20
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.3
+
+  cometbft-node2:
+    container_name: cometbft-node2
+    image: "cometbft/cometbft:v0.37.x"
+    environment:
+      - ID=2
+      - LOG=${LOG:-cometbft-node2.log}
+      - CMT_PEX=true
+      - CMT_PERSISTENT_PEERS="${CMT_PERSISTENT_PEERS}"
+      - CMT_PROXY_APP=tcp://fendermint-node2:26658
+    ports:
+      - "26661-26662:26656-26657"
+    volumes:
+      - ${HOME}/.ipc-testnet/node2/cometbft:/cometbft
+    healthcheck:
+      test: curl --fail http://localhost:26657 || exit 1
+      interval: 8s
+      timeout: 10s
+      retries: 20
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.4
+
+  cometbft-node3:
+    container_name: cometbft-node3
+    image: "cometbft/cometbft:v0.37.x"
+    environment:
+      - ID=3
+      - LOG=${LOG:-cometbft-node3.log}
+      - CMT_PERSISTENT_PEERS="${CMT_PERSISTENT_PEERS}"
+      - CMT_PROXY_APP=tcp://fendermint-node3:26658
+    ports:
+      - "26663-26664:26656-26657"
+    volumes:
+      - ${HOME}/.ipc-testnet/node3/cometbft:/cometbft
+    healthcheck:
+      test: curl --fail http://localhost:26657 || exit 1
+      interval: 8s
+      timeout: 10s
+      retries: 20
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.5
+
+  fendermint-node0:
+    container_name: fendermint-node0
+    image: "fendermint:latest"
+    build:
+      context: .
+    environment:
+      - FM_DATA_DIR=/data/fendermint/data
+      - FM_CHAIN_NAME=root
+      - LOG_LEVEL=info
+    volumes:
+      - ${HOME}/.ipc-testnet/node0:/data
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.6
+
+  fendermint-node1:
+    container_name: fendermint-node1
+    image: "fendermint:latest"
+    build:
+      context: .
+    environment:
+      - FM_DATA_DIR=/data/fendermint/data
+      - FM_CHAIN_NAME=root
```

Comment:
env var?

Reply:
Do you mean to set FM_CHAIN_NAME to env variable $CHAIN_NAME ?

Reply:
Anyway, I passed that variable

Reply:
Hm, sorry, I can't recall what I meant exactly. Probably that there exists an environment elsewhere already.

Reply:
Ok. I fixed that one